docs/op-guide/tune-tikv.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Tune TiKV Performance
+summary: Learn how to tune the TiKV parameters for optimal performance.
+category: operations
+---
+
+# Tune TiKV Performance
+
+This document describes how to tune the TiKV parameters for optimal performance.
+
+TiKV uses RocksDB for persistent storage at the bottom level of the TiKV architecture. Therefore, many of the performance parameters are related to RocksDB. TiKV uses two RocksDB instances: the default RocksDB instance stores KV data, the Raft RocksDB instance (RaftDB) stores Raft logs.
+
+TiKV implements `Column Families` (CF) from RocksDB.
+
+- The default RocksDB instance stores KV data in the `default`, `write` and `lock` CFs.
+
+    - The `default` CF stores the actual data. The corresponding parameters are in `[rocksdb.defaultcf]`.
+    - The `write` CF stores the version information in Multi-Version Concurrency Control (MVCC) and index-related data. The corresponding parameters are in `[rocksdb.writecf]`.
+    - The `lock` CF stores the lock information. The system uses the default parameters.
+
+- The Raft RocksDB (RaftDB) instance stores Raft logs.
+    
+    - The `default` CF stores the Raft log. The corresponding parameters are in `[raftdb.defaultcf]`.
+
+Each CF has a separate `block cache` to cache data blocks to accelerate the data reading speed in RocksDB. You can configure the size of the `block cache` by setting the `block-cache-size` parameter. The bigger the `block-cache-size`, the more hot data can be cached, and the easier to read data, in the meantime, more system memory will be occupied.
+
+Each CF also has a separate `write buffer`. You can configure its size by setting the `write-buffer-size` parameter.
+
+## Parameters specification
+
+```toml
+# Log level: trace, debug, info, warn, error, off.
+log-level = "info"
+
+[server]
+# Set listening address
+addr = "127.0.0.1:20160"
+
+# It is recommended to use the default value.
+notify-capacity = 40960
+messages-per-tick = 4096
+
+# Size of thread pool for gRPC
+grpc-concurrency = 4
+# The number of gRPC connections between each TiKV instance
+grpc-raft-conn-num = 10
+
+# Most read requests from TiDB are sent to the coprocessor of TiKV. This parameter is used
+# to set the number of threads of the coprocessor. If many read requests exist, add the
+# number of threads and keep the number within that of the system CPU cores. For example,
+# for a 32-core machine deployed with TiKV, you can even set this parameter to 30 in
+# repeatable read scenarios. If this parameter is not set, TiKV automatically sets it to
+# CPU cores * 0.8.
+end-point-concurrency = 8
```

Comment:
seem this is changed now. @breeswish

Reply:
OK. Then close this PR? @breeswish Could you help rewrite this doc?

Reply:
Yes, I will share it to you once complete.

Reply:
@breeswish Thanks!